data/human/variants.txt

```diff
@@ -81,6 +81,45 @@ ship "Argosy" "Argosy (Turret)"
 		"Hyperdrive"
 
 
+ship "Argosy" "Argosy (Plasma Repeater)"
+	outfits
+		"Anti-Missile Turret"
+		"D23-QP Shield Generator"
+		"Greyhound Plasma Thruster"
+		Hyperdrive
+		"Impala Plasma Steering"
+		"LP072a Battery Pack"
+		"Liquid Nitrogen Cooler"
+		"Outfits Expansion"
+		"Plasma Repeater" 2
+		"RT-I Radiothermal"
+
+
+ship "Argosy" "Argosy (Mega Blaster)"
+	outfits
+		"Blaster Turret" 2
+		"D23-QP Shield Generator"
+		"Greyhound Plasma Thruster"
+		Hyperdrive
+		"Impala Plasma Steering"
+		"LP072a Battery Pack"
+		"RT-I Radiothermal"
+		"Twin Blaster" 4
```

Comment:
I think this should be formatted to how other ships are done, iirc it's

```
 <guns>
 <turrets>
 <secondaries>

 <power>
 <systems>

 <engines>
 <ftl drive>
```
(Look at other ships to confirm again)

Same with every other variants.

Reply:
Yeah, I just used the all content plugin to make these ships and then took the ships from my player file. I'll go and rearrange them.